website/blog/2025-12-05-kubernetes-ai-conformance-aks/index.md

```diff
@@ -0,0 +1,180 @@
+
+### Observability
+
+#### Accelerator performance metrics
+
+Observability is critical for AI workloads. Platforms must expose GPU utilization, memory consumption, temperature, power draw, and other accelerator metrics through standard endpoints. You can deploy the [NVIDIA DCGM exporter](https://learn.microsoft.com/azure/aks/monitor-gpu-metrics) on AKS to expose GPU metrics in Prometheus format, and use [GPU health monitoring](https://learn.microsoft.com/azure/aks/gpu-health-monitoring) for proactive issue detection with Node Problem Detector (NPD) in your AI pipelines.
+
+#### AI service metrics
+
+Platforms must provide a monitoring system capable of discovering and collecting metrics from workloads that expose them in standard formats like Prometheus. AKS integrates with [Azure Monitor for containers](https://learn.microsoft.com/azure/azure-monitor/containers/kubernetes-monitoring-enable) and provides [Container Insights GPU monitoring](https://learn.microsoft.com/azure/azure-monitor/containers/container-insights-gpu-monitoring) for observability needs in your environment. For AI inference workloads, you can [monitor and visualize vLLM inference metrics](https://learn.microsoft.com/azure/aks/ai-toolchain-operator-monitoring) with Azure Managed Prometheus and Azure Managed Grafana when using the AI toolchain operator (KAITO) add-on.
+
+### Security
+
+#### Secure accelerator access
+
+Access to accelerators from within containers must be properly isolated and mediated by the Kubernetes resource management framework (device plugin or DRA) and container runtime, preventing unauthorized access or interference between workloads. See the [AKS secure accelerator access evidence](https://github.com/Azure/AKS/tree/master/ai-conformance/v1.34/secure_accelerator_access) for implementation details.
+
+### Operators
+
+#### AI operator support
+
+Modern AI platforms rely on Kubernetes operators to manage complex resources like training jobs, model servers, and distributed training coordinators. AKS supports the [AI Toolchain Operator (KAITO)](https://learn.microsoft.com/azure/aks/ai-toolchain-operator) to run inferencing, fine-tuning, and retrieval augmented generation (RAG). You can also install and run operators with custom resource definitions, such as [Ray on AKS](https://learn.microsoft.com/azure/aks/ray-overview).
+
+## Creating an AI-conformant AKS cluster
+
+To create an AI-conformant AKS cluster, be sure to choose a certified Kubernetes version with the appropriate features enabled.
+
+### Step 1. Create an AKS cluster
+
+AKS achieved AI Conformance certification starting with Kubernetes 1.34. Create an AKS cluster running Kubernetes 1.34 or later:
+
+```bash
+az aks create \
+  --resource-group <resource-group> \
+  --name <cluster-name> \
+  --enable-azure-monitor-metrics \

```

Comment:
IMO this should be an optional step similar to advance routing but for if you need AI service metrics. or at a minimum update the description to `Create an AKS cluster with Prometheus monitoring enabled.`